<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>822</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>视频监控异常处理系统</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="open_button">
          <property name="text">
           <string>打开摄像头(&amp;C)</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="shoot_button">
          <property name="text">
           <string>拍照(&amp;S)</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="record_button">
          <property name="text">
           <string>录像(&amp;R)</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="monitor_button">
          <property name="text">
           <string>开始监控(&amp;M)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="visual_radio_1">
          <property name="text">
           <string>input</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="visual_radio_2">
          <property name="text">
           <string>frame_diff</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="visual_radio_3">
          <property name="text">
           <string>motion_hist</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="visual_radio_4">
          <property name="text">
           <string>grad_orient</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="threshold_label">
          <property name="text">
           <string>阈值</string>
          </property>
          <property name="buddy">
           <cstring>threshold_spin</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="threshold_spin">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>255</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="threshold_slider">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksAbove</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="record_check">
          <property name="text">
           <string>录像</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="shoot_check">
          <property name="text">
           <string>拍照</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="shoot_delay_label">
          <property name="text">
           <string>拍照间隔</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="shoot_delay_spin">
          <property name="suffix">
           <string> 秒</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="shoot_delay_slider">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>10</number>
          </property>
          <property name="pageStep">
           <number>1</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksAbove</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="sound_check">
          <property name="text">
           <string>播放警报</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="exit_button">
          <property name="text">
           <string>退出(&amp;X)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="screen">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>640</width>
          <height>480</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string>1</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>822</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="file_menu">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="exit_action"/>
   </widget>
   <widget class="QMenu" name="tool_menu">
    <property name="title">
     <string>工具(&amp;T)</string>
    </property>
    <addaction name="open_action"/>
    <addaction name="shoot_action"/>
    <addaction name="record_action"/>
   </widget>
   <addaction name="file_menu"/>
   <addaction name="tool_menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="open_action">
   <property name="text">
    <string>打开摄像头(&amp;C)</string>
   </property>
  </action>
  <action name="shoot_action">
   <property name="text">
    <string>拍照(&amp;S)</string>
   </property>
  </action>
  <action name="record_action">
   <property name="text">
    <string>录像(&amp;R)</string>
   </property>
  </action>
  <action name="exit_action">
   <property name="text">
    <string>退出(&amp;X)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>exit_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>572</y>
    </hint>
    <hint type="destinationlabel">
     <x>670</x>
     <y>461</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>threshold_spin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>threshold_slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>366</y>
    </hint>
    <hint type="destinationlabel">
     <x>82</x>
     <y>391</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>threshold_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>threshold_spin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>61</x>
     <y>391</y>
    </hint>
    <hint type="destinationlabel">
     <x>64</x>
     <y>366</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>shoot_delay_spin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>shoot_delay_slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>37</x>
     <y>454</y>
    </hint>
    <hint type="destinationlabel">
     <x>32</x>
     <y>478</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>shoot_delay_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>shoot_delay_spin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>480</y>
    </hint>
    <hint type="destinationlabel">
     <x>50</x>
     <y>451</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
